src/js/popup.js

```diff
@@ -85,10 +85,21 @@ function showNagMaybe() {
         _showNag();
       }
     });
+
   } else if (POPUP_DATA.criticalError) {
```

Comment:
The error takes precedence over seenComic here. Is that intended? If a user installs PB and their disk is low on space, the badger icon will show an "!" but when the they click on the icon, they'll only see the comic nag. They need to close the popup and open it again to see the error message.

What if we have the comic nag on top, and the error popup underneath? That way a recurring error that the user ignores doesn't block the comic nag, but the user won't get confused by a lack of error message.

Reply:
This sounds like a good idea, although perhaps we should just ignore non-Privacy Badger related messages altogether.